lom/build-deprecated.sh

```diff
@@ -61,7 +61,9 @@ dbClientTest() {
 }
 
 test_plugin_mgr() {
-    go test -v -p 1 -cover $1 -coverprofile=coverprofile_plmgr.out -coverpkg lom/src/lib/lomcommon,lom/src/pluginmgr/pluginmgr_common,lom/src/pluginmgr/plugins_common,lom/src/pluginmgr/plugins_files -covermode=atomic ./src/pluginmgr/pluginmgr_test ./src/pluginmgr/pluginmgr_common
+    #CGO_ENABLED=1 GORACE="log_path=/tmp/race/gou_report"  go test -race -v -p 1 -cover $1 -coverprofile=coverprofile_plmgr.out -coverpkg lom/src/pluginmgr/pluginmgr_common,lom/src/plugins/plugins_common,lom/src/plugins/plugins_files -covermode=atomic ./src/pluginmgr/pluginmgr_common 
```

Comment:
@renukamanavalan  ..I use this file for any experiment commands instead of polluting  makefile  like for e.g. gorace detection, static analysis commands, etc. . We can keep for sometime until final check-in.

Reply:
make it available in makefile as special target and delete this file

Reply:
TODO